### ISSUE TITLE
predator nes tile colors

### ISSUE THREAD

**Issue by revility** [spiiin/CadEditor#7]
When replacing tiles in predator for nes, the colors do not change to the new tile replacing it.  Everything else works fine and the tile functions as intended,  but the colors do not update.

example:  If you replace one of the blue rocks with with the brown grass, it turns into blue grass when running the game.

**Comment by spiiin** [comment]
Yes, pallete bytes encoded in separate layer for this game, but configs not support it now. I'll work on fix this

**Comment by revility** [comment]
Thanks!  It's not a great game, but with some level modding and sprite replacements, which is easy to do,  it could turn into something decent & become the go to version of it which I started to do. :)
